rotkehlchen/premium/premium.py

```diff
@@ -296,12 +296,19 @@ def query_last_data_metadata(self) -> RemoteMetadata:
             raise RemoteError(msg) from e
 
         result = _process_dict_response(response)
-        metadata = RemoteMetadata(
-            upload_ts=Timestamp(result['upload_ts']),
-            last_modify_ts=Timestamp(result['last_modify_ts']),
-            data_hash=result['data_hash'],
-            data_size=result['data_size'],
-        )
+
+        try:
+            metadata = RemoteMetadata(
+                upload_ts=Timestamp(result['upload_ts']),
+                last_modify_ts=Timestamp(result['last_modify_ts']),
+                data_hash=result['data_hash'],
+                data_size=result['data_size'],
+            )
+        except KeyError as e:
+            msg = f'Problem connecting to rotki server. last_data_metadata response missing {e!s} key'  # noqa: E501
+            log.error(f'{msg}. Response was {result}')
```

Comment:
Also here the error is silent for the user. The only thing that will be noticed is that the account doesn't have the premium status

Reply:
> Also here the error is silent for the user. The only thing that will be noticed is that the account doesn't have the premium status

that's how all these errors are so far handled by rotki no? All failures to connec to rotki server.